### ISSUE TITLE
[BUG] My Canvas Courses Not Displaying an Upcoming Summer Course

### ISSUE THREAD

**Issue by vburgett** [rokwire/illinois-app#4136]
On both dev (6.0.8, Android) and prod (5.1.34, iPhone), all of my Canvas courses are displaying the Illinois app except for my upcoming summer course. I do see this summer course already in the Canvas app.

I expected to see this summer course in the Illinois app too.

![Screenshot 2024-06-18 at 5 16 40 PM](https://github.com/rokwire/illinois-app/assets/1044905/d768a041-a730-400b-9d33-58ec2e0e9472)

**Comment by reynlds-illinois** [comment]
I was able to reproduce this with your account, and several others that have a Summer, 2024, enrollment. The call (using python/requests) is only returning 10 records at a time, and it appears that alpha sorting is causing some to fall off the display. Increasing the records returned using "params" brings back all enrollments. Example:

```
    params = {'per_page': 100}
    r = requests.get(url, headers=headers, params=params).json()
```
It's definitely working as quite a few enrollments are coming back outside of course/term dates with the correct "error":

`    {'access_restricted_by_date': True, 'id': 24586}`

Is there a way to show "more" in the app, such as adding a scroll to it? I'm not an app dev and I don't use the Illinois app at all. This may be a referral to Instructure Dev Support/Consulting.

**Comment by vburgett** [comment]
Thank you, Mark, for chiming in and testing this with my account and others.

@dobromirdobrev can we return more records at a time and order the classes based on the most recently added courses or the term they are associated with (Summer 2024, Fall 2024, etc)?

**Comment by dobromirdobrev** [comment]
@reynlds-illinois  thank you for your input! It was helpful.

@vburgett , this issue required changes on both Illinois app and backend side (LMS BB). The changes in the Illinois app are available in version 6.0.17. The relevant backend issue is [LMS 112](https://github.com/rokwire/lms-building-block/issues/112). Once it gets resolved, you can verify it on dev.

As a result - the course from the last screenshot should be visible and the courses will be ordered by created date (e.g. the most recent are at the top).

**Comment by vburgett** [comment]
That worked in 6.0.17 on dev. Thank you, @dobromirdobrev!
